app/routes.py

```diff
@@ -0,0 +1,269 @@
+    db.session.delete(video)
+    db.session.commit()
+    return {"id": video.id}, 200
+
+
+# --------------------------------
+# ----------- RENTALS ----------
+# --------------------------------
+
+
+# CHECK OUT A VIDEO TO A CUSTOMER
+@rentals_bp.route("/check-out", methods=["POST"])
+def checkout_video():
+    request_body = request.get_json()
+    if "customer_id" not in request_body:
+        return {"details": "Request body must include customer_id."}, 400
+    elif "video_id" not in request_body:
+        return {"details": "Request body must include video_id."}, 400
+    customer_id = request_body["customer_id"]
+    video_id = request_body["video_id"]
+    customer = Customer.query.get(customer_id)
+    video = Video.query.get(video_id)
+    # check if customer and video exist
+    if customer is None or video is None:
+        return make_response("", 404)
+    num_of_videos_rented = Rental.query.filter_by(
+        video_id=video.id, checked_out=True).count()
+    available_inventory = video.total_inventory - num_of_videos_rented
+    if available_inventory == 0:
+        return jsonify({"message": f"Could not perform checkout"}), 400
+    new_rental = Rental(customer_id=customer.id,
+                        video_id=video.id,
+                        due_date=(datetime.now() + timedelta(days=7)),
+                        checked_out=True)
+    db.session.add(new_rental)
+    db.session.commit()
+    videos_checkout_by_customer = Rental.query.filter_by(
+        customer_id=customer.id, checked_out=True).count()
+    available_inventory -= 1
+    return jsonify({
+        "customer_id": new_rental.customer_id,
+        "video_id": new_rental.video_id,
+        "due_date": new_rental.due_date,
+        "videos_checked_out_count": videos_checkout_by_customer, "available_inventory": available_inventory}), 200
+
+
+# CHECK IN A VIDEO TO A CUSTOMER
+@rentals_bp.route("/check-in", methods=["POST"])
+def checkin_video():
+    request_body = request.get_json()
+    if "customer_id" not in request_body:
+        return {"details": "Request body must include customer_id."}, 400
+    elif "video_id" not in request_body:
+        return {"details": "Request body must include video_id."}, 400
+    customer_id = request_body["customer_id"]
+    video_id = request_body["video_id"]
+    customer = Customer.query.get(customer_id)
+    video = Video.query.get(video_id)
+    # check if customer and video exist
+    if customer is None or video is None:
+        return make_response("", 404)
+    rental = Rental.query.filter_by(
+        video_id=video.id, customer_id=customer.id, checked_out=True).first()
+    if rental is None:
+        return jsonify({"message": f"No outstanding rentals for customer {customer.id} and video {video.id}"}), 400
+    rental.checked_out = False
+    db.session.commit()
+    num_of_videos_rented = Rental.query.filter_by(
+        video_id=video.id, checked_out=True).count()
```

Comment:
I do like your use of count() when you just need the number of records. This saves the database from having to send back the full data of the rows, and the database itself is very good at counting rows.